requirements.txt

```diff
@@ -0,0 +1 @@
+-e .
```

Comment:
I'd like to copy pytest on this I think https://github.com/pytest-dev/pytest/blob/master/requirements-docs.txt